src/util/analytics-client.ts

```diff
@@ -12,8 +12,8 @@ let analyticsNodeClient;
 
 export function sendAnalytics(eventType: "screen", attributes?: { name: string } & Record<string, unknown>)
```

Comment:
Should `attributes?: { name: string }` be `attributes: { name: string }`? Remove the `?` to match the removal of `!` in the code bellow?